cmd/imaginator/main.go

```diff
@@ -47,6 +47,10 @@ func main() {
 		os.Exit(1)
 	}
 	logger := serverlogger.New("")
+	if umask := syscall.Umask(022); umask != 022 {
```

Comment:
`syscall.Umask` never fails but it returns the previous value. Why does it matter for imaginator what the old value was?